web-admin/src/routes/[organization]/[project]/explore/[dashboard]/+page.svelte

```diff
@@ -76,11 +76,25 @@
     });
   }
 
+  // Display a dashboard banner
+  $: if ($explore.data?.explore?.explore?.state?.validSpec?.banner) {
```

Comment:
Nit: since you're repeating this twice, it'd be marginally neater to pull it out as an explanatory variable:
```
$: hasBanner = !!$explore.data?.explore?.explore?.state?.validSpec?.banner
```

Then also, the below `$explore...banner` does not need optional chaining.